index.js

```diff
@@ -436,6 +436,65 @@ const waitOnMaybe = () => {
 
 const I = (x) => x
 
+const detectPrNumber = async () => {
+  const {
+    GITHUB_SHA,
+    GITHUB_TOKEN,
+    GITHUB_RUN_ID,
+    GITHUB_REPOSITORY,
+    GITHUB_HEAD_REF,
+    GITHUB_REF,
+    GITHUB_SERVER_URL,
+    CYPRESS_PULL_REQUEST_ID,
+    CYPRESS_PULL_REQUEST_URL
+  } = process.env
+
```

Comment:
Because we're not overriding the existing env, we can abort early if both are already set:

```js
if (CYPRESS_PULL_REQUEST_ID && CYPRESS_PULL_REQUEST_URL) {
  // Both pull request envs are already defined
  return 
}
```

Reply:
Ahh good call! I'll add this 👍